mautic/mautic
# Add support for tokens for SMS

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | No
| New feature/enhancement? (use the a.x branch)      | Yes
| Deprecations?                          | No
| BC breaks? (use the c.x branch)        | No
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This PR added support for suggestion tokens similar like you can use in builder

![image](https://user-images.githubusercontent.com/462477/150675196-4f5bd3e7-3889-400f-9df0-f33a75273256.png)




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. On production env need `mautic:assets:generate`
3.  On SMS text message field you are able to use `{` + CTRL SPACE to autosuggestion of tokens, included {ownerfield} tokens
4. Then send SMS and see If tokens were replace correctly




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10779"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
